modules/api/pages/using-list-and-search-methods.adoc

```diff
@@ -51,6 +51,34 @@ sob.or();
 sob.differentFrom</b>(HumanTaskInstanceSearchDescriptor.ASSIGNEE_ID, myUser.getId());
 ----
 
+
+[#case_sensitivity]
+== Case sensitivity
+
+WARNING: On Oracle database, search is case-sensitive.
+
+All Search requests done using **search terms** are case-insensitive but **filtering** IS case-sensitive.
```

Comment:
It might be because i'm not familiar with the subject, but I'm not sure to understand when I have case-sensitiveness and when I don't: 

- On the warning above, you say that 'On oracle, **search** is case sensitive'
- Then you say that search requests using search terms are case-insensitive but filtering is case sensitive

It's a bit confusing and definitively not straight forward, you start your explanation with an exception and  then you state a rule. 
I suggest you to remove the warning an to simply write two sentences: 
- On most of the database, ...
- On Orcale, ...

Reply:
I agree. I'll change that!